PennMobile/build.gradle

```diff
@@ -53,6 +53,9 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8
+    }
```

Comment:
is this needed?

Reply:
Yes, I added it in order to support the viewmodel initialization using `by activityViewModel()` in order to share the viewmodel across the fragments within PCA, referenced in `PennCourseAlertCreateAlertFragment:36`